### ISSUE TITLE
Uninitialized constant Net::ReadTimeout (NameError)

### ISSUE THREAD

**Issue by filingroove** [atipugin/telegram-bot-ruby#3]
ruby 1.9.3p484 (2013-11-22 revision 43786) [x86_64-linux]
on Elementary OS Freya

Getting "uninitialized constant Net::ReadTimeout (NameError)" error on my script startup

Full message reads:
```
/var/lib/gems/1.9.1/gems/telegram-bot-ruby-0.3.0/lib/telegram/bot/client.rb:34:in `rescue in listen': uninitialized constant Net::ReadTimeout (NameError)
	from /var/lib/gems/1.9.1/gems/telegram-bot-ruby-0.3.0/lib/telegram/bot/client.rb:23:in `listen'
	from find.rb:16:in `block in <main>'
	from /var/lib/gems/1.9.1/gems/telegram-bot-ruby-0.3.0/lib/telegram/bot/client.rb:19:in `run'
	from /var/lib/gems/1.9.1/gems/telegram-bot-ruby-0.3.0/lib/telegram/bot/client.rb:7:in `run'
	from find.rb:15:in `<main>'
```

**Comment by atipugin** [comment]
Thanks! It happens because Net::ReadTimeout isn't defined in Ruby 1.9.x (2.x only). I'll make it compatible with older versions asap.

**Comment by atipugin** [comment]
I've just released 0.3.1 with fix for this issue. Please give it a try.
